crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -211,7 +211,22 @@ where
         }
```

Comment:
we need to apply this here

what we should here is:
```rust
let header = if let Some(pending) = self.client().pending_header()? {
    pending
} else {
  let mut latest = self.client().latest_header();
  // modify the latest values

}
// use existing code to fill cfg, block_env

```

Reply:
I understand, made some changes. Let me know what you think! Thanks for the feedback.

Reply:
I'm a bit confused here `latest_header()` isn't a method for the client.